packages/client/hmi-client/src/components/workflow/tera-simulate-node.vue

```diff
@@ -0,0 +1,242 @@
+
+		// start polling for run status
+		getStatus();
+	}
+};
+
+// watch for changes in node input
+// watch(
+// 	() => props.node.inputs,
+// 	async (inputList) => {
+// 		const forecastOutputList = await Promise.all(
+// 			inputList.map(({ value }) =>
+// 				makeForecast({
+// 					model: value.model.id,
+// 					initials: value.initialValues,
+// 					params: value.parameterValues,
+// 					tspan: [0.0, 90.0] // hardcoded timespan
+// 				})
+// 			)
+// 		);
+// 		startedRunIdList.value = forecastOutputList.map((forecastOutput) => forecastOutput.id);
+//
+// 		// start polling for run status
+// 		getStatus();
+// 	},
+// 	{ deep: true }
+// );
+
+// Retrieve run ids
+// FIXME: Replace with API.poller
+const getStatus = async () => {
+	const currentRunStatus = await Promise.all(startedRunIdList.value.map(getRunStatus));
+
+	if (currentRunStatus.every(({ status }) => status === 'done')) {
+		completedRunIdList.value = startedRunIdList.value;
+	} else if (currentRunStatus.some(({ status }) => status === 'queuing')) {
+		// recursively call until all runs retrieved
+		setTimeout(getStatus, 3000);
+	} else {
+		// throw if there are any failed runs for now
+		console.error('Failed', startedRunIdList.value);
+		throw Error('Failed Runs');
+	}
+};
+
+watch(
+	() => completedRunIdList.value,
+	async (runIdList) => {
+		const newRunResults = {};
+		await Promise.all(
+			runIdList.map(async (runId) => {
+				if (runResults[runId]) {
+					newRunResults[runId] = runResults[runId];
+				} else {
+					const resultCsv = await getRunResult(runId);
+					newRunResults[runId] = csvParse(resultCsv);
+				}
+			})
+		);
+		runResults = newRunResults;
+		// process data retrieved
+
+		// assume that the state variables for all runs will be identical
+		// take first run and parse it for state variables
+		if (!stateVariablesList.length) {
+			stateVariablesList = Object.keys(runResults[Object.keys(runResults)[0]][0])
+				.filter((key) => key !== 'timestep')
+				.map((key) => ({ code: key }));
+		}
+		selectedVariable.value = [stateVariablesList[0]];
+		runList = runIdList.map((runId, index) => ({ code: runId, index }));
+		selectedRun.value = runList[0];
+	}
+);
+
+const renderGraph = ([selectedVarList]) => {
+	const datasets: any[] = [];
+	selectedVarList.forEach(({ code }) =>
+		completedRunIdList.value
+			.map((runId) => runResults[runId])
+			.forEach((run, runIdx) => {
+				const dataset = {
+					data: run.map(
+						(datum: any) => datum[code] // - runResults[selectedRun.value.code][timeIdx][code]
+					),
+					label: `${completedRunIdList.value[runIdx]} - ${code}`,
+					fill: false,
+					tension: 0.4
+				};
+				datasets.push(dataset);
+			})
+	);
+	chartData.value = {
+		labels: runResults[Object.keys(runResults)[0]].map((datum) => Number(datum.timestep)),
+		datasets
+	};
+};
+watch(() => [selectedVariable.value, selectedRun.value] as any, renderGraph);
+</script>
+
+<style scoped>
+.result-container {
```

Comment:
The `<style>` is scoped, so you do not need this class, just use `section`
```suggestion
section {
```